MeshLib/CoordinateSystem.h

```diff
@@ -0,0 +1,100 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#ifndef COORDINATESYSTEMTYPE_H_
+#define COORDINATESYSTEMTYPE_H_
+
+#include <cmath>
+
+#include "GeoLib/AABB.h"
+
+namespace MeshLib
+{
+
+class Element;
+
+/**
+ * \brief Coordinate system type
+ */
+struct CoordinateSystemType
+{
+    enum type
```

Comment:
i initially tried to use it but changed it to this way  because the code is shorter and simpler as I don't need to make cast (from enum class to int) each time whenever bit operation is used.